### DISCUSSION TITLE
Prevent expiration notifications for stopped containers

### DISCUSSION THREAD

**Discussion by sey** [nginx-proxy/acme-companion#1115] [Ideas]
This is a question more than a bug report.  

When I stop a container with a `LETSENCRYPT_HOST`, I keep receiving expiration notifications from LetsEncrypt until the certificate is actually expired. 
It is not a big issue but it would be great if the certificate of a stopped container was revoked automatically if there are no other containers with this domain. 

I couldn't find an existing issue or documentation related to this topic.

**Comment by buchdag** [comment]
To be honest, I doubt this will ever be done because of the amount of work it would require and the potential issues that will inevitably pop up if we start automating certificate revocation based on container status, for very marginal QoL gain.

I'd advise to switch to ZeroSSL instead, they provide a web admin interface where you can easily revoke certificates.
